Marlin/ubl.cpp

```diff
@@ -47,7 +47,7 @@
     safe_delay(10);
   }
 
-  static void serial_echo_10x_spaces() {
+  static void serial_echo_11x_spaces() {
     for (uint8_t i = GRID_MAX_POINTS_X - 1; --i;) {
       SERIAL_ECHOPGM("          ");
```

Comment:
Nope...  That is supposed to be 12x.   But I lost some data during a check in.   The internet died halfway through a Sync and I couldn't use the fork any more.   I thought I got all my changes reconstructed...   But I missed that one.   

It turns out that needs to be 12 spaces to look right with the proportional fonts.